cleanlab/datalab/internal/issue_manager/underperforming_group.py

```diff
@@ -71,8 +71,10 @@ class UnderperformingGroupIssueManager(IssueManager):
         1: [],
         2: ["threshold"],
     }
-    OUTLIER_CLUSTER_LABELS = (-1,)
-    NO_UNDERPERFORMING_CLUSTER_ID = min(OUTLIER_CLUSTER_LABELS) - 1
+    OUTLIER_CLUSTER_LABELS: ClassVar[Tuple[int]] = (-1,)
+    """Specifies labels considered as outliers by the clustering algorithm."""
+    NO_UNDERPERFORMING_CLUSTER_ID: ClassVar[int] = min(OUTLIER_CLUSTER_LABELS) - 1
+    """Constant to signify absence of any underperforming cluster."""
```

Comment:
[nit] use # instead of """ for in-code comments

Reply:
Using #, the description of class vars don't show up in the documentation. 

With #
![image](https://github.com/cleanlab/cleanlab/assets/23188723/05f2c5cc-187e-4b09-bdee-8874f885c48a)

With """
![image](https://github.com/cleanlab/cleanlab/assets/23188723/22e21cc4-03ad-41a8-a022-5e4f20fe8398)

Reply:
ah sorry disregard me then!